contracts/instadapp/AnteInstadappTreasuryBalanceTest.sol

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.8.0;
+
+import {AnteTest} from "../AnteTest.sol";
+import {IERC20Metadata} from "@openzeppelin/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+
+/// @title Instadapp Treasury Token Balance Test
+/// @notice Ante Test to check if Instadapp Treasury maintains a total balance greater than
+///          2 million INST tokens
+contract AnteInstadappTreasuryBalanceTest is AnteTest("Instadapp Treasury Balance greater than 2 million") {
+    
+    address public constant tokenAddr = 0x6f40d4A6237C257fff2dB00FA0510DeEECd303eb;
+    address public constant treasuryAddr = 0x28849D2b63fA8D361e5fc15cB8aBB13019884d09;
+
+    IERC20Metadata public INST = IERC20Metadata(tokenAddr);
+
+    uint256 public immutable thresholdBalance;
+    uint256 public immutable decimals;
+
+    constructor() {
+        protocolName = "Instadapp";
+        testedContracts = [treasuryAddr];
+        decimals = 6;
```

Comment:
I see the decimals being 18 here: https://etherscan.io/address/0x6f40d4A6237C257fff2dB00FA0510DeEECd303eb#readContract

Is there a reason why we want it to be 6 decimal places?

Reply:
out of habit. Now I think it's better to use all decimals. I'll edit and commit again